.travis/utils.sh

```diff
@@ -18,6 +18,10 @@ setup_kafka() {
     nohup bash -c "cd kafka && bin/kafka-server-start.sh config/server.properties &"
     sleep 10
     kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic case --zookeeper localhost:2181
+    kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic form --zookeeper localhost:2181
+    kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic meta --zookeeper localhost:2181
+    kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic sql-case --zookeeper localhost:2181
+    kafka/bin/kafka-topics.sh --create --partitions 1 --replication-factor 1 --topic sql-form --zookeeper localhost:2181
```

Comment:
this doesn't feel totally sustainable. i think it's the case that the first request per-topic fails, so i think the app layer could be updated to just try twice (can look into that later if it seems like a good idea)